src/app/remotes/help-item-editor/help-item-editor.component.spec.ts

```diff
@@ -71,6 +71,10 @@ describe('OneCXHelpItemEditorComponent', () => {
         {
           provide: BASE_URL,
           useValue: baseUrlSubject
+        },
+        {
+          provide: HAS_PERMISSION_CHECKER,
```

Comment:
Maybe it would be better to make the UserService the default PermissionChecker. So the IfPermissionDirective would inject both, the PermissionChecker (as optional) and the UserService and would use the UserService if no checker is provided